### PR TITLE
Front end/login

### PR DESCRIPTION
**Pull request by divyashah1**
FrontEnd Login uing JWT

### CODE REVIEW

File: HRS_FRONTEND/src/app/attendence/attendence.component.html [hunk 1]
```diff
@@ -1,35 +1,67 @@
 <table class="table table-hover">
     <thead>
```
Comment (nirav-prakash) [inline HRS_FRONTEND/src/app/attendence/attendence.component.html:2]: Why it's a fixed header ?

File: HRS_FRONTEND/src/app/attendence/attendence.component.ts [hunk 2]
```diff
@@ -2,31 +2,40 @@ import { Component } from '@angular/core';
...
   ngOnInit(): void {
     this.read(); //for getall
 
   }
 
   read() {
-    this.ApiService.getall().subscribe(
-      data => {
-        this.attendance = data;
-        console.log(data)
```
Comment (nirav-prakash) [inline HRS_FRONTEND/src/app/attendence/attendence.component.ts:28]: console ?

File: HRS_FRONTEND/src/app/auth-guard.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
```
Comment (nirav-prakash) [inline HRS_FRONTEND/src/app/auth-guard.service.spec.ts:1]: TestBed ?

**Review by nirav-prakash — CHANGES_REQUESTED** [review]
Why Attendance related changes in Login PR ?

File: HRS_FRONTEND/src/app/auth-guard.service.spec.ts (hunk 3 above)
Comment (divyashah1) [reply HRS_FRONTEND/src/app/auth-guard.service.spec.ts:1]: sir this is default file for testing

**Pull request closed without merging**